pyaedt/edb_core/stackup.py

```diff
@@ -621,6 +621,35 @@ def export_stackup(self, fpath, file_format="xml", include_material_with_layer=F
             self._logger.warning("Layer stackup format is not supported. Skipping import.")
             return False
 
+    @pyaedt_function_handler
+    def export_stackup(self, fpath, file_format="xml", include_material_with_layer=False):
+        """Export stackup definition to a CSV or JSON file.
+
+        .. deprecated:: 0.6.61
+           Use :func:`export` instead.
+
+        Parameters
+        ----------
+        fpath : str
+            File path to csv or json file.
+        file_format : str, optional
+            Format of the file to export. The default is ``"csv"``. Options are ``"csv"``, ``"xlsx"``,
+            ``"json"``.
+        include_material_with_layer : bool, optional.
+            Whether to include the material definition inside layer ones. This parameter is only used
```

Comment:
Here and in line 641, can you replace the pronoun "ones" with the noun. Is it "inside layer definitions"? I was unable to determine what the pronoun referred to.